docs/workflows/_sdk-setup/_passkey-management/_deleting-a-passkey-flutter.mdx

```diff
@@ -0,0 +1,7 @@
+The `deletePasskey` function allows you to delete a passkey given its ID.
```

Comment:
nit: the text for the various functions all various functions seems to be repeated . You can probably move that to right under the header - in this case `### Deleting a Passkey`

Reply:
I'm debating this. I see I can do that with "Bind Passkey URL Validation", "Authenticate URL Validation",
"Listing Passkeys" and "Deleting a Passkey", but I can't do that with "Setup", "Binding a Passkey" or "Authentication", at least as they're currently written. I might at least wait until further review.

Reply:
It looks like we should be able to do this for "Binding a Passkey" and "Authentication". There are some differences written under JS but the info should be the same info for all the SDKs when calling these functions so we can consolidate the differences. If we remove the text inside these then the code snippets can be reused in other documents like possibly the ones Jen is writing. It'll also make it easier to make updates to text if it's only in one place.

Reply:
I agree that reusable code snippet files would be ideal, but I am not sure yet if these should be those files. I suppose they can be, but a couple of these contain both the request and the response, which I think I should split out first.

However, I notice we still have some inconsistencies. For example, `BindPasskeyResponse` has a url which depending on the sdk is either `postBindRedirect` (JavaScript), `postBindingRedirectUri` (Android, Flutter) or `postBindingRedirectURI` (Swift, React Native). I wonder if this (any/all differences in requests/responses) should be one more item for 2.0.0 before we release.

Reply:
Ok, this should be done.